pkg/rancher-components/src/components/Form/Radio/RadioButton.vue

```diff
@@ -134,7 +144,7 @@ export default Vue.extend({
     @click.stop="clicked($event)"
   >
     <input
-      :id="_uid+'-radio'"
+      :id="randomString"
```

Comment:
I think the intention is to invoke the randomString method here. Right now the function is getting set as the ID. 

I think it would be better to have a data property used here so that the id stays consistent if the component is re-rendered because of a :key change or something like that.

Reply:
Good catch, that's not quite intended at all 🙂

> I think it would be better to have a data property used here so that the id stays consistent if the component is re-rendered because of a :key change or something like that.

I think handling it this way will be an improvement overall, thanks for the suggestion!